std/process.d

```diff
@@ -4210,3 +4211,16 @@ else version (Posix)
 }
 else
     static assert(0, "os not supported");
+
+version (Windows) { /* Doesn't use BROWSER */ }
+else
+@system unittest
+{
+    import std.conv : text;
+    import std.range : repeat;
+    immutable string url = text("http://", repeat('x', 249));
+
+    TestScript prog = `if [ "$1" != "` ~ url ~ `"; then exit 1; fi`;
```

Comment:
Missing ` ]` before `;`; so the test apparently doesn't fail correctly at the moment... :)

Reply:
Good catch. The problem is that `browse` doesn't wait for the forked process but returns immediatly...

I could add smth like this:

```d
version (StdUnittest)
{
    int status;
    const check = waitpid(childpid, &status, 0);
    assert(check != -1);
    assert(status == 0);
}
```